content/tutorials/manage-the-rippled-server/configuration/connect-your-rippled-to-the-xrp-test-net.md

```diff
@@ -80,6 +85,11 @@ Edit your `rippled.cfg` file.
         [network_id]
         20
 
+    *AMM-Devnet*
+
+        [network_id]
+        25
```

Comment:
tangent / just curious: how was this number chosen?

Reply:
I have no idea. I think the NFT-Devnet used 20 because of XLS-20d, but the AMM proposal is XLS-30d so that doesn't apply here.